.checkov.baseline

```diff
@@ -66,7 +66,8 @@
                 {
                     "resource": "AWS::IAM::ManagedPolicy.PivotRolepolicy3",
                     "check_ids": [
-                        "CKV_AWS_109"
+                        "CKV_AWS_109",
```

Comment:
just tested and re-running chekcov command and creating a new baseline will likely remove this change ... would wait until we finalize pivotRole policies though before we run it one last time before PR merge

Reply:
I believe when re-running checkov command and creating a new baseline this exception on `PivotRolepolicy3` is no longer present